Xamarin.Forms.Platform.UAP/SwitchRenderer.cs

```diff
@@ -53,5 +69,54 @@ void UpdateFlowDirection()
 		{
 			Control.UpdateFlowDirection(Element);
 		}
+
+		void UpdateOnColor()
+		{
+			if (Control == null)
+				return;
+
+			var grid = Control.GetChildren<Windows.UI.Xaml.Controls.Grid>().FirstOrDefault();
+			var groups = Windows.UI.Xaml.VisualStateManager.GetVisualStateGroups(grid);
+			foreach (var group in groups)
+			{
+				if (group.Name != "CommonStates")
+					continue;
+
+				foreach (var state in group.States)
+				{
+					if (state.Name != "PointerOver")
+						continue;
+
+					foreach (var timeline in state.Storyboard.Children.OfType<ObjectAnimationUsingKeyFrames>())
+					{
+						var property = Storyboard.GetTargetProperty(timeline);
+						var target = Storyboard.GetTargetName(timeline);
+						if ((target == "SwitchKnobBounds") && (property == "Fill"))
+						{
+							var frame = timeline.KeyFrames.First();
+
+							if (_originalOnColor == null)
+								_originalOnColor = frame.Value;
+
+							if (Element.IsSet(Switch.OnColorProperty) && !Element.OnColor.IsDefault)
```

Comment:
The additional `IsDefault` check is needed (unfortunately) since just using `IsSet` when setting `Color.Default` as a value, would still return `true` causing the control to disappear

Reply:
the `color.IsDefault` is probably enough

Reply:
Then I'm confused. In another place, I had to use `IsSet`. What is the use-case for one or the other?